<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FCS_IoT Patch Tool v.1.0</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>381</width>
     <height>34</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="btn_back">
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>&lt; Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_patch">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Patch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>701</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../qrc/100.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="text">
    <string>Modify Whitelist</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Modify whitelist app allowed to install</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>531</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="title">
    <string>Whitelist:</string>
   </property>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pub">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Select app to add or remove.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>The app with the same signatures as below would be installed, 
 but the rest of them would be not.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_apk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Caution:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Without any signature, no app has access to device.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_install">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>with Install</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>listWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>splitter</zorder>
  <zorder>label_logo</zorder>
  <zorder>label_6</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>chk_install</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
